docs/concepts.md

```diff
@@ -197,25 +197,20 @@ compares both confidentiality and integrity:
 - the **integrity** check ensures that the value sent is trusted by the
   destination
 
-More concretely, if `L_a = (C_a, I_a)` (where `C_a` and `I_a` are the
-confidentiality and integrity components respectively), and `L_b = (C_b, I_b)`
-then `L_a ⊑ L_b` iff `C_a ⊆ C_b` and `I_a ⊇ I_b`. Notably, the integrity part of
-the check is "flipped" by convention (see "Integrity Considerations for Secure
-Computer Systems" below). More fundamental than adhering to convention, by
-flipping the integrity order in this way, we can retain the intuitive meaning
-behind integrity tags as representing trust by a principal (trust by a user,
-trust on a subject, etc.).
+Intuitively, data can only flow from a source to a destination if:
 
-Intuitively, data can only flow from `a` to `b` if:
+- the destination is **at least as secret** as the source
+- the source is **at least as trusted** as the destination
 
-- `b` is **at least as secret** as `a`
-- `a` is **at least as trusted** as `b`
+> Notably, the integrity part of the check is "flipped" by convention (see
+> "Integrity Considerations for Secure Computer Systems" below). More
+> fundamental than adhering to convention, by flipping the integrity order in
+> this way, we can retain the intuitive meaning behind integrity tags as
+> representing trust by a principal (trust by a user, trust on a subject, etc.).
```

Comment:
Is this intentionally quoted? Seems odd, since it's not really a quote AFAICT?

Reply:
Happy to make it normal text, if preferred.

It's a formatting convention used by a few docs I've worked on before, essentially marks text that can be classified as "further/more-in-depth/explanatory/aside" type information. Agree it's odd that it uses markdown quotes for formatting though.

Reply:
I would keep it as normal text for now, especially since AFAICT there is no precedent in our documentation, so as it is it actually stands out even more than normal text. :)

Reply:
fair enough :)